source/fundamentals/query-data.txt

```diff
@@ -169,9 +169,105 @@ on the ``name`` field.
    When sorting on fields with a boolean value, entities with a field value of
    ``false`` show before those with a value of ``true``.
 
+Scalar Aggregations
+-------------------
+
+The {+provider-short+} supports the following scalar aggregation methods:
+
+- ``Count()``: Returns the number of elements in a collection
+- ``LongCount()``: Returns the number of elements in a collection as a ``long``
```

Comment:
Same as Count we can count matches where a predicate is true for a given document.